src/main/kotlin/domain/rank/Rank.kt

```diff
@@ -9,11 +9,11 @@ enum class Rank(val countOfMatch: Int, val winningMoney: Int) {
     MISS(0, 0);
 
     companion object {
-        fun valueOf(countOfMatch: Int, matchBonus: Boolean): Rank =
-            if (checkRankIsThird(countOfMatch, matchBonus)) Rank.THIRD else
-                Rank.values().find {
-                    it.countOfMatch == countOfMatch
-                } ?: Rank.MISS
+        fun valueOf(countOfMatch: Int, matchBonus: Boolean): Rank {
+            if (checkRankIsThird(countOfMatch, matchBonus))
+                return THIRD
```

Comment:
```suggestion
            if (checkRankIsThird(countOfMatch, matchBonus)) {
                return THIRD
            }
```
코틀린에서는 중괄호를 넣어주는 것이 컨벤션입니다.

ref. https://kotlinlang.org/docs/coding-conventions.html#control-flow-statements